java/arcs/android/util/AndroidLog.kt

```diff
@@ -13,22 +13,25 @@ package arcs.android.util
 
 import arcs.core.util.Log
 
+private val TAG = "Arcs"
+
 /** Initializes [Log] for tests on the JVM. */
-fun initLogForAndroid(level: Log.Level = mapAndroidLogLevel("Arcs")) {
+fun initLogForAndroid(level: Log.Level = mapAndroidLogLevel(TAG)) {
```

Comment:
I think we should still retain at least an `Arcs` prefix on the logcat tag.

Reply:
Even with this change we do, don't we? My intent was just to move it to a constant so we can have it in one place for here and in the mapping below.